casper/src/main/scala/coop/rchain/casper/genesis/contracts/Rev.scala

```diff
@@ -0,0 +1,26 @@
+package coop.rchain.rholang.genesis.contracts
+
+import coop.rchain.casper.util.rholang.InterpreterUtil
+import coop.rchain.rholang.build.CompiledRholangSource
+import coop.rchain.models.Par
+
+final class Rev(wallets: Seq[Wallet]) extends CompiledRholangSource {
+  val code = s"""
+    |//requires MakeMint, BasicWallet
+    |new revMintCh in {
+    |  @"MakeMint"!(*revMintCh) | for(@revMint <- revMintCh) {
+    |    //TODO: How should the revMint unforgeable name be exposed (if at all)?
+    |
+    |    //public contract for making empty rev purses
+    |    contract @["Rev", "makePurse"](return) = {
+    |       @[revMint, "makePurse"]!(0, *return)
+    |    } |
+    |
+    |    //basic wallets which exist from genesis
+    |    ${wallets.map(Wallet.rhoCode).mkString(" |\n")}
+    |  }
+    |}
+  """.stripMargin
```

Comment:
Can we put these snippets of code in resources instead of embedding them in Scala code?

Reply:
It uses Scala String Interpolation, so must be in the Scala code.

Reply:
All pure Rholang is separately given in the `src/main/rholang` and `src/test/rholang` directories